packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/delete/routing.js

```diff
@@ -0,0 +1,35 @@
+import { TRACKING_HIT_PREFIX } from './constants';
+import { TRACKING_SUFFIX } from '../../constants';
+import { TRACKING_NAME } from '../../../constants';
+import { deletePoolResolve } from '../../components/delete/utils';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(
+    'octavia-load-balancer.loadbalancer.pools.detail.general-information.delete',
+    {
+      url: '/delete?poolId&poolName',
+      views: {
+        modal: {
+          component: 'octaviaLoadBalancerPoolsDelete',
+        },
+      },
+      layout: 'modal',
+      resolve: {
+        ...deletePoolResolve,
+        goBack: /* @ngInject */ ($state, trackAction) => (confirm) => {
+          if (!confirm) {
```

Comment:
This is incorrect, in the case an error arise when deleting the pool, you would pass through this if and track a cancel

Reply:
OK